evil-collection.el

```diff
@@ -326,6 +326,7 @@ See `evil-collection-init' and `evil-collection--modes-with-delayed-setup'."
     view
     vlf
     vterm
+    eat
```

Comment:
The list is in lexicographic order

Reply:
I have updated the list to adhere to the specified ordering requirements.